actions-server/src/server.ts

```diff
@@ -1,40 +1,40 @@
-const express = require("express");
-const bodyParser = require("body-parser");
-const Sentry = require("@sentry/node");
+import Sentry from "@sentry/node"
+import bodyParser from "body-parser"
+import express from "express"
 
 if (process.env.NODE_ENV !== "production") {
-  require("dotenv").config();
+  require("dotenv").config()
 }
 
 if (process.env.NODE_ENV !== "development") {
   Sentry.init({
     dsn:
       "https://593a557ed4834276803af5fc8a4432b5@o392843.ingest.sentry.io/5241108",
     environment: process.env.NODE_ENV,
-  });
+  })
 }
 
-const app = express();
+const app = express()
 
-const PORT = process.env.PORT || 3000;
+const PORT = process.env.PORT || 3000
 
-app.use(bodyParser.json());
+app.use(bodyParser.json())
 
 app.post("/:route", (req, res) => {
   try {
-    const handler = require(`./handlers/${req.params.route}`);
+    const handler = require(`./handlers/${req.params.route}`)
```

Comment:
there's probably a better way to do this sort of thing in TS. also kinda overkill, there is only 1 endpoint..